nifi-docker/dockerhub/sh/update_cluster_state_management.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh -e
+
+#    Licensed to the Apache Software Foundation (ASF) under one or more
+#    contributor license agreements.  See the NOTICE file distributed with
+#    this work for additional information regarding copyright ownership.
+#    The ASF licenses this file to You under the Apache License, Version 2.0
+#    (the "License"); you may not use this file except in compliance with
+#    the License.  You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+login_providers_file=${NIFI_HOME}/conf/state-management.xml
```

Comment:
should update this variable

Reply:
ok, I'll fix that. RE the compose file, got one already. You want me to check it into our repo or post it in Confluence?

Reply:
I like the idea of it being in the repo so when we add new features we can have it evolve to support those as well.  Maybe we can create another folder in nifi-docker for now and place it in there?  At some point, I hope to start migrating such efforts across the ecosystem to https://github.com/apache/nifi-container but haven't had the opportunity to do so.

Reply:
@apiri are there any jiras entered for nifi-container?

Reply:
@ottobackwards Only at a high level:  https://issues.apache.org/jira/browse/NIFI-4771

The issue is we have a lot of duplication and slight one offs for utility type functions that are pervasive throughout.  Would be good to get a more uniform handling of such resources across all convenience images we offer as a community.